### ISSUE TITLE
How to compute the OA and mIoU on the Semantickitti dataset

### ISSUE THREAD

**Issue by longmalongma** [yanx27/PointASNL#11]
How do you compute the OA and mIoU on the Semantickitti dataset? You don't seem to have published the code for it.

**Comment by yanx27** [comment]
@longmalongma Hi, when you submit to official semanticKITTI benchmark, it will generate output log for OA and mIoU.  Also, if you want to compute OA and mIoU for validation set, you can use official semanticKITTI api [here](https://github.com/PRBonn/semantic-kitti-api) just by following codes:
```
$ ./evaluate_semantics.py --dataset /path/to/kitti/dataset/ --predictions /path/to/method_predictions --split train/valid/test # depending of desired split to evaluate
```
By running `test_semantic_kitti_grid.py`, it will automatically generate predicted labels for evaluation.

**Comment by longmalongma** [comment]
> @longmalongma Hi, when you submit to official semanticKITTI benchmark, it will generate output log for OA and mIoU. Also, if you want to compute OA and mIoU for validation set, you can use official semanticKITTI api [here](https://github.com/PRBonn/semantic-kitti-api) just by following codes:
> 
> ```
> $ ./evaluate_semantics.py --dataset /path/to/kitti/dataset/ --predictions /path/to/method_predictions --split train/valid/test # depending of desired split to evaluate
> ```
> 
> By running `test_semantic_kitti_grid.py`, it will automatically generate predicted labels for evaluation.

Thanks for your reply, I see it!

**Comment by longmalongma** [comment]
> > @longmalongma Hi, when you submit to official semanticKITTI benchmark, it will generate output log for OA and mIoU. Also, if you want to compute OA and mIoU for validation set, you can use official semanticKITTI api [here](https://github.com/PRBonn/semantic-kitti-api) just by following codes:
> > ```
> > $ ./evaluate_semantics.py --dataset /path/to/kitti/dataset/ --predictions /path/to/method_predictions --split train/valid/test # depending of desired split to evaluate
> > ```
> > 
> > 
> > By running `test_semantic_kitti_grid.py`, it will automatically generate predicted labels for evaluation.
> 
> Thanks for your reply, I see it!

Can it compute OA and mIoU for test set???

**Comment by yanx27** [comment]
@longmalongma You need to submit on official benchmark of semanticKITTI

**Comment by longmalongma** [comment]
![Screenshot from 2020-06-21 12-00-42](https://user-images.githubusercontent.com/40291722/85216470-5aae9780-b3b7-11ea-9be0-4daabaf93bb4.png)


Thanks for your reply, can you tell me which files ｓhould I submit? only .label files from sequences 08,11 to 21? Can you give me the web site to refer to the predicted results？　Ｓorry, I am first use senmantickitti daatset.

**Comment by yanx27** [comment]
Hi, @longmalongma , sequence 08 only for validation set. If you want to submit to official semantickitti benchmark [here](https://competitions.codalab.org/competitions/20331#learn_the_details), you need to firstly generate sequence 11~21. Then zip them together and submit to the single scan competitions.

**Comment by longmalongma** [comment]
> Hi, @longmalongma , sequence 08 only for validation set. If you want to submit to official semantickitti benchmark [here](https://competitions.codalab.org/competitions/20331#learn_the_details), you need to firstly generate sequence 11~21. Then zip them together and submit to the single scan competitions.

Thanks, it only need .label files?
